screen-capture/idlharness.window.js

```diff
@@ -0,0 +1,16 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+// https://w3c.github.io/mediacapture-screen-share/
```

Comment:
Well, this is one of the more annoying discrepancies between TR and github.io I've seen :)